### PR TITLE
Linux Compatibility

### PR DESCRIPTION
**Pull request by NadieFiind**
Please don't merge yet. Wait for me to make it compatible with Linux first.

### CODE REVIEW

File: bin/pynani.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 except:
     pass
 
-os.system("cls")
+os.system("cls" if os.name == "nt" else "clear") # supports both Windows and Linux
```
Comment (NadieFiind) [inline bin/pynani.py:14]: I don't recommend clearing the terminal tho.

Comment (jabezborja) [reply bin/pynani.py:14]: You can remove it.

Comment (NadieFiind) [reply bin/pynani.py:14]: okay

**Comment by jabezborja** [comment]
Is this now ready to merge? Or I just leave it here?

**Review by jabezborja — APPROVED** [review]
LGTM!!!

File: README.md [hunk 2]
```diff
@@ -44,27 +43,51 @@ RunApp(MyApp())
...
-And now, you have the PyNani on your local machine!
+### Linux
+Get the source code from PyNani repo on GitHub.
+```bash
+git clone https://github.com/jabezborja/pynani.git -b stable
+```
+
+Make the [pynani.py](bin/pynani.py) an executable file.
+```bash
+chmod +x PyNani/bin/pynani.py
```
[the remark this answers is not in the document]
Comment (NadieFiind) [reply README.md:76]: In Linux, You need to give it permissions to become an executable file. Otherwise it will not work.

**Pull request merged**